core/data/src/test/java/com/google/samples/apps/nowinandroid/core/data/repository/OfflineFirstUserDataRepositoryTest.kt

```diff
@@ -129,4 +147,44 @@ class OfflineFirstUserDataRepositoryTest {
                     .first()
             )
         }
+
+    @Test
+    fun offlineFirstUserDataRepository_set_theme_brand_delegates_to_nia_preferences() =
```

Comment:
weird test naming, should have max two underscores

Reply:
This is consistent the existing (weird) test naming so I've filed a bug here to fix this to avoid polluting this PR.  https://github.com/android/nowinandroid/issues/345